config/projects.yaml

```diff
@@ -193,11 +193,7 @@ doctrine-extensions:
 doctrine-mongodb-admin-bundle:
     phpstan: true
     psalm: true
-    custom_gitignore_part: |
-        /tests/App/public/bundles
-    excluded_files:
-        - phpunit.xml.dist.twig
-        - tests/bootstrap.php.twig
```

Comment:
If I understand correctly, a PR is needed for doctrine-mongodb-admin-bundle, to add a custom_boostrap.

Reply:
Yes, but as it is now `custom_boostrap` instead of `custom_bootstrap` I was going to wait this to be deployed and I'll do it in that PR.